annotations/component-annotations/src/main/java/io/ap4k/component/handler/ComponentHandler.java

```diff
@@ -17,25 +17,37 @@
 
 import io.ap4k.Handler;
 import io.ap4k.Resources;
-import io.ap4k.component.config.CompositeConfig;
-import io.ap4k.component.config.EditableCompositeConfig;
+import io.ap4k.component.config.ComponentConfig;
+import io.ap4k.component.config.EditableComponentConfig;
+import io.ap4k.component.decorator.AddBuildConfigToComponentDecorator;
 import io.ap4k.component.decorator.AddEnvToComponentDecorator;
 import io.ap4k.component.decorator.AddRuntimeTypeToComponentDecorator;
 import io.ap4k.component.decorator.AddRuntimeVersionToComponentDecorator;
 import io.ap4k.component.model.Component;
 import io.ap4k.component.model.ComponentBuilder;
-import io.ap4k.component.model.ComponentFluent;
 import io.ap4k.kubernetes.config.ConfigKey;
 import io.ap4k.kubernetes.config.Configuration;
 import io.ap4k.kubernetes.config.Env;
 import io.ap4k.utils.Strings;
+import org.eclipse.jgit.lib.Repository;
```

Comment:
We want to avoid having any dependency to 3rd party libraries (only exception is compile only dependencies). Everything else is shaded into our big fat `dependencies` jar.

Please add and shade jgit into https://github.com/ap4k/dependencies, and then use it here.

Reply:
There is a compilation problem. ap4k-dependencies 0.4-SNAPSHOT uses kubernetes-client 4.2.2 and this class `io.ap4k.deps.kubernetes.api.model.rbac.KubernetesRoleBindingBuilder` used in ap4k is not present in the jar. I have to investigate more.

Reply:
@aureamunoz: no need to investigate. The `Kubernetes` prefix has been removed.

I am about to create a PR and update to latest dependencies.